tests/integration/ci-runner.py

```diff
@@ -393,11 +393,23 @@ def _get_all_tests(self, repo_path):
             )
         )
 
+        # Avoid overlaps with previous runs
+        logging.info("Clearing dmesg before run")
+        subprocess.check_call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
+            "dmesg --clear", shell=True
+        )
+
         logging.info("Getting all tests with cmd '%s'", cmd)
         subprocess.check_call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
```

Comment:
Stop, but here we just getting all tests. Let's dump dmesg after tests run.

Reply:
Indeed, good catch. Fixed.